src/krb5/_creds.pyi

```diff
@@ -25,6 +44,9 @@ class Creds:
         context: Krb5 context.
     """
 
+    @property
+    def addr(self) -> int:
```

Comment:
Did you add this for a reason? AFAIK this isn't exposed on the actual `Creds` object.

Reply:
Hmm. I thought it was exposed. Probably I added it at some point to the `.pyx` file and then removed it again.

I'll remove it from the `.pyi` file.